### PR TITLE
Comment on generated-code property .

### PR DESCRIPTION
**Pull request by arnaudruffin**
Added comment in LICENSE.txt and generated classes headers, as discussed in issue #1356.
I just specified that work including aa-generated code could be distributed under any license. Feel free to edit, of course.

### CODE REVIEW

File: AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/generation/CodeModelGenerator.java [hunk 1]
```diff
@@ -26,11 +26,12 @@
 public class CodeModelGenerator {
 
 	private final Filer filer;
-	private final String aaVersion;
+	private final String header;
 
 	public CodeModelGenerator(Filer filer, String aaVersion) {
 		this.filer = filer;
-		this.aaVersion = aaVersion;
+        this.header = "DO NOT EDIT THIS FILE, IT HAS BEEN GENERATED USING AndroidAnnotations " + aaVersion + ".\n " +
```
Comment (WonderCsabo) [inline AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/generation/CodeModelGenerator.java]: Maybe another \n here to separate the two unrelated statements?

File: LICENSE.txt [hunk 2]
```diff
@@ -31,3 +31,7 @@ product.
 
 We are therefore allowed to distribute CodeModel without Modification as
 part of AndroidAnnotations.
+
+About AndroidAnnotations generated files: if you create a larger work that 
+contains files generated by AndroidAnnotation, you can distribute that work
```
Comment (WonderCsabo) [inline LICENSE.txt]: AndroidAnnotation**s**

**Comment by WonderCsabo** [comment]
Thanks for the PR!

You have [Checkstyle violations](https://travis-ci.org/excilys/androidannotations/jobs/56407886#L1021).

Can you amend the commits per my comments?

**Comment by arnaudruffin** [comment]
I am on it :)

**Comment by WonderCsabo** [comment]
Please squash the commits into one.

**Comment by arnaudruffin** [comment]
Sorry about that. I am no Git expert. My --amend didn't work as expected but thanks to that I improved my "rebase" skills

**Comment by WonderCsabo** [comment]
Thanks for the PR!

**Pull request merged**